src/index.html

```diff
@@ -249,4 +252,12 @@
                 src="assets/dash/2x/3-1 - aot compile@2x-80.jpg"/>
         </div>
     </section>
+    <section class="row dash-row">
+        <div class="dash-dartpad">
+            <h2>Try Dart in your browser!</h2>
+            <select id="dartpad-select"></select>
+            <div id="dartpad-host"></div>
+            <h3>Another call to action</h3>
```

Comment:
@kwalrath do we have the words ready yet?

Reply:
Sorry, I didn't realize you were waiting on me. I'll work on the words.

In the meantime, could you modify the "Try Dart" link(s?) to go to the embedded dartpad?

Reply:
How about something like this:

Want more practice? <a href="/codelabs">Try a codelab.</a> Or <a href="/get-dart">download Dart.</a>

Reply:
/cc @messi01, @galeyang for the text/approach.

@johnpryan if we use an h3, we'll need to fix the link color.